test/com/facebook/buck/testrunner/testdata/simple_testng_setup/test/SimpleTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-present Facebook, Inc.
```

Comment:
I don't know Addepar's procedures for using open-source code in our code base but thought I'd flag this as a possible Bad Thing (TM). Or maybe it's fine.